### PR TITLE
[FEAT] Land Expansion Boosts

### PR DESCRIPTION
**Pull request by mGranTY**
# Description

*ADDED PARSNIP TOOL BOOST
*ADDED AMULETS BOOST
*ADDED SUNFLOWER SHIELD BOOST
*ADDED BUMPKIN SKILL "MASTER FARMER" BOOST 
*FIXED TREE RECOVERY TIME IF THE USER HAS A BEAVER + THE TREE HUGGER SKILL

## Type of change

Please delete options that are not relevant.

- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Added new .tests
Add skill master farmer skill to your bumpkin and harvest anything
harvest a parsnip with the bumpkin using the parsnip tool
harvest a beetroot using the beetroot amulet in your bumpkin

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [X] I have read the contributing guidelines and agree to the T&Cs
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/events/landExpansion/plant.test.ts
Comment (Rofrtd) [inline src/features/game/events/landExpansion/plant.test.ts:414]: We might not need this commented line here.

File: src/features/game/events/landExpansion/plant.test.ts
Comment (Rofrtd) [inline src/features/game/events/landExpansion/plant.test.ts:451]: This one as well.

File: src/features/game/events/landExpansion/plant.ts
Comment (Rofrtd) [inline src/features/game/events/landExpansion/plant.ts:185]: Looks like we're not checking for `Green Thumb` skills here as well. Could you add it please?

```
  if (skills["Green Thumb"]) {
    amount *= 1.05;
  }
```

**Review by Rofrtd — COMMENTED** [review]
Looking good, just left a couple of comments.

File: src/features/game/events/landExpansion/plant.test.ts
Comment (mGranTY) [reply src/features/game/events/landExpansion/plant.test.ts]: Fixed.

File: src/features/game/events/landExpansion/plant.test.ts
Comment (mGranTY) [reply src/features/game/events/landExpansion/plant.test.ts]: Fixed.

File: src/features/game/events/landExpansion/plant.ts
Comment (mGranTY) [reply src/features/game/events/landExpansion/plant.ts:185]: Done :)

File: src/features/game/events/landExpansion/feedBumpkin.ts
Comment (mGranTY) [inline src/features/game/events/landExpansion/feedBumpkin.ts:42]: This will probably conflict with https://github.com/sunflower-land/sunflower-land/pull/1693 

So i will wait until that PR gets merged to fix it.

Comment (mGranTY) [reply src/features/game/events/landExpansion/feedBumpkin.ts]: Conflicts solved!

File: src/features/game/types/crops.ts
Comment (Rofrtd) [inline src/features/game/types/crops.ts:107]: These won't be here, they are fruits and we'll add them later on.

**Review by Rofrtd — APPROVED** [review]
Great work.

LGTM

**Pull request merged**
